lib/dal/src/component.rs

```diff
@@ -2089,41 +2089,6 @@ impl Component {
             .clone_attributes_from(ctx, self.id(), true, true)
             .await?;
 
-        // Enqueue creation actions
```

Comment:
this is what was causing the duplicate creates - we were enqueuing a create action in both the paste AND on creating the component

Reply:
Great catch! I totally missed that